API/Services/Implementations/PlayerStatsService.cs

```diff
@@ -274,7 +274,7 @@ DateTime dateMax
             // TODO: Different way of calcing this
             // AverageMatchCostAggregate = ratingStats.Average(x => x.MatchCost),
             HighestRating = adjustments.Max(x => x.RatingAfter),
-            RatingGained = adjustments.Last().RatingAfter - adjustments.First().RatingAfter,
+            RatingGained = adjustments.Last().RatingAfter - adjustments.FirstOrDefault()?.RatingAfter ?? 0,
```

Comment:
Probably bad to use `Last()` over `LastOrDefault()` anywhere (we just fixed that one crash with `First()`). That, and since there is no certainty of sorting here assuming the last adjustment is the most recent is optimistic... This approach probably more consistent:
```cs
RatingGained = adjustments.Sum(ra => ra.RatingDelta),
```
`RatingGained` should probably be renamed to `RatingDelta` or something similar as well

Reply:
Renaming would make this a breaking change so would rather not do that yet. But yes sum of rating delta sounds like it will achieve what we want.